### ISSUE TITLE
I want to add some most useful algorithms code in C++ for--

### ISSUE THREAD

**Issue by Bharath-KumarReddy** [mr-ayush-agrawal/Hacktoberfest#31]
1) K-way merge Algorithm

2) Stack Reduced Quick Sort

3) Merge sort combined with Insertion Sort

4) Prims, Topological, Kruskal's algorithm

5) DijKstra , Floyd Warshall ,bellman ford algorithm

6) Backtracking
a) Rat Maze
b) N-Queen
c) Sudoku
d) Sum of Subset

7) Activity selection , Huffman Encoding

please consider this under Hacktoberfest_2023 as I want to contribute myself : )

**Comment by mr-ayush-agrawal** [comment]
Yes You can go for this :)

**Comment by Bharath-KumarReddy** [comment]
Yes Sure I had just done this , and I made a Pull request once Please check : )

**Comment by itssiddhantjain** [comment]
I want to add some most useful algorithms code in C++. @Bharath-KumarReddy please consider this under Hacktoberfest_2023 as I want to contribute myself : )

**Comment by mr-ayush-agrawal** [comment]
Hi @ADHIRAJ-12 
I will review and update you asap.

**Comment by mr-ayush-agrawal** [comment]
@itssiddhantjain You are free to make Contributions anytime in this repo :)

**Comment by ADHIRAJ-12** [comment]
Thanks @mr-ayush-agrawal
